ball.ai/src/main/java/com/google/sps/data/Game.java

```diff
@@ -0,0 +1,3 @@
+package com.google.sps.data;
+
+public class Game { public String day, team1, team2; }
```

Comment:
No class methods? What is this going to do?

Reply:
Just there for my CSV reader to read data into